### DISCUSSION TITLE
tuiTextfield does not respect readonly attribute

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9873] [Q&A] [answered]
Hi, cannot find in https://taiga-ui.dev/components/input example with readonly attribute but doing this

```
<input tuiTextfield  readonly />
```

leaves input field editable. However if you remove tuiTextfield input becomes readonly.

Do you have any example how to attchieve readonly or this is a bug and should be fixed?

**Comment by waterplea** [comment, marked as answer]
Use `[readOnly]="true"`
